spec/controllers/spotlight/catalog_controller_spec.rb

```diff
@@ -22,6 +22,8 @@
         uploaded_resource.save_and_index
       end
 
+      sleep 10
```

Comment:
@corylown I tried the sleep 10 in multiple spots of the code and it didn't work.